src/app/TargetSelect/TargetSelect.tsx

```diff
@@ -251,11 +263,24 @@ export const TargetSelect: React.FunctionComponent<TargetSelectProps> = (props)
 
   return (
     <>
-      <Card>
-        <CardHeader>
-          <CardHeaderMain>
+      <Card isExpanded={props.simple || selected === NO_TARGET ? undefined : isExpanded}>
+        <CardHeader
+          onExpand={props.simple || selected === NO_TARGET ? undefined : onExpand}
```

Comment:
Since these props are `undefined` if `props.simple || selected === NO_TARGET`, maybe we could just defined a memo object that contains these props and spread its content out if not `props.simple || selected === NO_TARGET`?

Reply:
On this note, how about this? Probably easier to look at all the needed props.

```typescript
const cardHeaderProps = React.useMemo(() => props.simple? {}: (
    {
      onExpand: onExpand,
      isToggleRightAligned: true,
      toggleButtonProps: {
        id: 'target-select-expand-button',
        'aria-label': 'Details',
        'aria-labelledby': 'expandable-card-title target-select-expand-button',
        'aria-expanded': isExpanded,
      }
    }
  ), [onExpand, props.simple, isExpanded]);

  return (
    <>
      <Card isExpanded={props.simple ? undefined : isExpanded}>
        <CardHeader
          {...cardHeaderProps}
        >
[...to be continued]
```

What do you think?